services/movie_session.py

```diff
@@ -42,3 +42,8 @@
 
 def delete_movie_session_by_id(session_id: int) -> None:
     MovieSession.objects.get(id=session_id).delete()
+
+
+def get_taken_seats(movie_session_id: MovieSession) -> QuerySet:
```

Comment:
The parameter `movie_session_id` should be of type `int`, not `MovieSession`. Update the type hint accordingly.